tests/lib/rules/manifest-exists/tests.ts

```diff
@@ -9,62 +9,62 @@ const htmlWithManifestSpecified = generateHTMLPage(`<link rel="manifest" href="s
 
 const tests: Array<IRuleTest> = [
     {
-        name: `Web app manifest is not specified`,
-        reports: [{ message: 'Web app manifest not specified' }],
+        name: `Manifest is not specified`,
+        reports: [{ message: 'Manifest not specified' }],
         serverConfig: generateHTMLPage('<link rel="stylesheet" href="style.css">')
     },
     {
-        name: `Web app manifest is specified multiple times`,
-        reports: [{ message: 'Web app manifest already specified' }],
+        name: `Manifest is specified multiple times`,
+        reports: [{ message: 'Manifest already specified' }],
         serverConfig: {
             '/': generateHTMLPage(`<link rel="manifest" href="site1.webmanifest"><link rel="manifest" href="site2.webmanifest">`),
             '/site1.webmanifest': '',
             '/site2.webmanifest': ''
         }
     },
     {
-        name: `Web app manifest is specified with no 'href'`,
-        reports: [{ message: `Web app manifest specified with invalid 'href'` }],
+        name: `Manifest is specified with no 'href'`,
+        reports: [{ message: `Manifest specified with invalid 'href'` }],
         serverConfig: generateHTMLPage('<link rel="manifest">')
     },
     {
-        name: `Web app manifest is specified with empty 'href'`,
-        reports: [{ message: `Web app manifest specified with invalid 'href'` }],
+        name: `Manifest is specified with empty 'href'`,
+        reports: [{ message: `Manifest specified with invalid 'href'` }],
         serverConfig: generateHTMLPage('<link rel="manifest" href="">')
     },
     {
-        name: `Web app manifest is specified as a full URL`,
+        name: `Manifest is specified as a full URL`,
         serverConfig: {
             '/': generateHTMLPage('<link rel="manifest" href="http://localhost/site.webmanifest">'),
             '/site.webmanifest': ''
         }
     },
     {
-        name: `Web app manifest is specified and the file exists`,
+        name: `Manifest is specified and the file exists`,
         serverConfig: {
             '/': htmlWithManifestSpecified,
             '/site.webmanifest': ''
         }
     },
     {
-        name: `Web app manifest is specified and request for file fails`,
-        reports: [{ message: `Web app manifest file request failed` }],
+        name: `Manifest is specified and request for file fails`,
```

Comment:
we should also say in the test name that we expect to pass in this case (and for similar ones where fetching the manifest fails).

Reply:
Done.